modules/ossm-migrating-premigration-checklists-using-cert-manager.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-premigration-checklists-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-premigration-checklists-using-cert-manager_{context}"]
+= Check if you are using the cert-manager tool with your deployment
+
+It is important to know if you are using the cert-manager tool with {SMProduct} 2. There are a series of configuration steps you must perform before you can start migrating your deployments and workloads.
+
+.Procedure
+
+. Use one of the following ways to check if you are using the cert-manager tool with your deployment:
+
+.. Inspect your `ServiceMeshControlPlane` resource:
++
+[source,yaml]
+----
+spec:
+  security:
+    certificateAuthority:
+      ...
+      type: cert-manager
+----
+
+.. Run the following command:
```

Comment:
```suggestion
.. Run the following command to ... :
```
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command